providers/apis/defi/uniswapv3/fetcher.go

```diff
@@ -110,7 +110,7 @@ func (u *PriceFetcher) Fetch(
 	for i, ticker := range tickers {
 		pool, err := u.GetPool(ticker)
 		if err != nil {
-			u.logger.Error(
```

Comment:
Similar comment as for raydium--it seems like we should have some error log (only 1 though) when we fail outright. Maybe we still will though?

Reply:
here - we will have an error log, since we're returning the error.  Eventually it will be logged